types/abstract-things/thing.d.ts

```diff
@@ -0,0 +1,34 @@
+import EventEmitter from './events';
+
+export interface EmitEventOptions {
+    multiple: boolean;
+}
+
+export class Thing {
+    constructor(...args: any);
+    new(...input: any[]): any;
+
+    get id(): string | null;
+    set id(identifier: string | null);
+
+    init(): Promise<undefined | this>;
+    initCallback(): Promise<void>;
+
+    destroy(): Promise<void>;
+    destroyCallback(): Promise<void>;
+
+    emitEvent(event: string, data: unknown, options?: EmitEventOptions): void;
```

Comment:
```suggestion
    emitEvent(event: string, data?: unknown, options?: EmitEventOptions): void;
```
the param is optional IMO:
https://github.com/thingbound/abstract-things/blob/440404b3b7f72047ad154f49f1f289eeda736232/thing.js#L72